sql/src/main/java/org/apache/druid/sql/calcite/schema/SystemSchema.java

```diff
@@ -234,29 +236,29 @@ public SystemSchema(
       final CoordinatorClient coordinatorClient,
       final OverlordClient overlordClient,
       final DruidNodeDiscoveryProvider druidNodeDiscoveryProvider,
-      final ObjectMapper jsonMapper
+      final ObjectMapper jsonMapper,
+      final @EscalatedClient HttpClient httpClient
   )
   {
     Preconditions.checkNotNull(serverView, "serverView");
-    this.tableMap = ImmutableMap.of(
-        SEGMENTS_TABLE,
-        new SegmentsTable(druidSchema, metadataView, jsonMapper, authorizerMapper),
-        SERVERS_TABLE,
-        new ServersTable(
-            druidNodeDiscoveryProvider,
-            serverInventoryView,
-            authorizerMapper,
-            overlordClient,
-            coordinatorClient,
-            jsonMapper
-        ),
-        SERVER_SEGMENTS_TABLE,
-        new ServerSegmentsTable(serverView, authorizerMapper),
-        TASKS_TABLE,
-        new TasksTable(overlordClient, authorizerMapper),
-        SUPERVISOR_TABLE,
-        new SupervisorsTable(overlordClient, authorizerMapper)
-    );
+    this.tableMap = ImmutableMap.<String, Table>builder()
```

Comment:
Please use the original syntax `ImmutableMap.of()` and reduce the diff size here.